### PR TITLE
Add la_forge: a pulsar timing array bayesian data visualization package

### PR DESCRIPTION
**Pull request by AaronDJohnson**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AaronDJohnson** [comment]
@Hazboun6 are you okay with being a maintainer on this one?

**Comment by AaronDJohnson** [comment]
@conda-forge/help-python I think this one is ready for review.

**Comment by AaronDJohnson** [comment]
@conda-forge/help-python this is ready for review

File: recipes/la_forge/meta.yaml
Comment (BastianZim) [inline recipes/la_forge/meta.yaml:23]:
```suggestion
    - numpy >=1.11
```

https://github.com/nanograv/la_forge/blob/e167b0c17b8721e5e383d0fc96ca175e0f3e0c78/setup.py#L14

File: recipes/la_forge/meta.yaml
Comment (BastianZim) [inline recipes/la_forge/meta.yaml:24]: Upstream says `>1.0.0`. I assume that this one here is correct but it would be good to have this upstream as well.

https://github.com/nanograv/la_forge/blob/e167b0c17b8721e5e383d0fc96ca175e0f3e0c78/setup.py#L15

File: recipes/la_forge/meta.yaml
Comment (BastianZim) [inline recipes/la_forge/meta.yaml:25]:
```suggestion
    - matplotlib-base >=2.0.0
```

https://github.com/nanograv/la_forge/blob/e167b0c17b8721e5e383d0fc96ca175e0f3e0c78/setup.py#L16

File: recipes/la_forge/meta.yaml
Comment (BastianZim) [inline recipes/la_forge/meta.yaml:28]: I cannot find this upstream.

File: recipes/la_forge/meta.yaml
Comment (BastianZim) [inline recipes/la_forge/meta.yaml:50]:
```suggestion
  dev_url: https://github.com/nanograv/la_forge
```

Just to make it consistent.

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of things, otherwise this is ready to go.

File: recipes/la_forge/meta.yaml
Comment (AaronDJohnson) [reply recipes/la_forge/meta.yaml:24]: Upstream has been fixed (main branch). This will carry over to PyPI tomorrow.

File: recipes/la_forge/meta.yaml
Comment (AaronDJohnson) [reply recipes/la_forge/meta.yaml:28]: I see these in setup.py. Although matplotlib instead of matplotlib-base. Is there somewhere else that I should put them?

Comment (AaronDJohnson) [reply recipes/la_forge/meta.yaml:28]: I have now double checked that they are there (at least on the main branch). Again, we'll get it on PyPI tomorrow.

Comment (BastianZim) [reply recipes/la_forge/meta.yaml:28]: Ahh yes, you‘re right. I was looking at `main` - my bad.

**Review by BastianZim — APPROVED** [review]

**Comment by BastianZim** [comment]
This looks great. Just waiting for CI to finish.

**Comment by BastianZim** [comment]
OSX is an internal server error. But since the build phase went through we can ignore that.

**Pull request merged**
